build-tools/xaprepare/xaprepare/ConfigAndData/Dependencies/AndroidToolchain.cs

```diff
@@ -38,13 +38,16 @@ public AndroidToolchain ()
 				new AndroidPlatformComponent ("platform-25_r03",   apiLevel: "25", pkgRevision: "3"),
 				new AndroidPlatformComponent ("platform-26_r02",   apiLevel: "26", pkgRevision: "2"),
 				new AndroidPlatformComponent ("platform-27_r03",   apiLevel: "27", pkgRevision: "3"),
-				new AndroidPlatformComponent ("platform-28_r04",   apiLevel: "28", pkgRevision: "4"),
-				new AndroidPlatformComponent ("platform-29_r01",   apiLevel: "29", pkgRevision: "1"),
-				new AndroidPlatformComponent ("platform-R_r02",    apiLevel: "R",  pkgRevision: "2"),
+				new AndroidPlatformComponent ("platform-28_r06",   apiLevel: "28", pkgRevision: "6"),
+				new AndroidPlatformComponent ("platform-29_r04",   apiLevel: "29", pkgRevision: "4"),
+				new AndroidPlatformComponent ("platform-R_r03",    apiLevel: "R",  pkgRevision: "3"),
```

Comment:
Let's *not* update API-R to Developer Preview 3, please.  This should be done separately, as there's no guarantee that it's API compatible with DP2.